tensorflow/lite/delegates/gpu/BUILD

```diff
@@ -88,7 +88,9 @@ objc_library(
 # build -c opt --config android_arm64 --copt -Os --copt -DTFLITE_GPU_BINARY_RELEASE --copt -fvisibility=hidden --linkopt -s --strip always :libtensorflowlite_gpu_gl.so
 cc_binary(
     name = "libtensorflowlite_gpu_gl.so",
-    linkopts = select({
+    linkopts = [
+        "-Wl,-soname=libtensorflowlite_gpu_gl.so"
```

Comment:
buildifier failed because you forgot the trailing comma, i.e. the line should be:

"-Wl,-soname=libtensorflowlite_gpu_gl.so",

Note the trailing comma

Reply:
Added the trailing comma.  In the future, is there a linter for bazel changes?